src/components/TooltipTrigger.js

```diff
@@ -21,6 +21,7 @@ const styles = {
     borderColor: colors.secondary(),
     padding: '1rem',
     boxShadow: '0 1px 3px 2px rgba(0,0,0,0.3)',
+    lineHeight: '1.5em',
```

Comment:
I tried and failed to see the impact of this on the PR site. It might be worth an extra screenshot or two since this component is used so widely.

Reply:
Added screenshots - we COULD apply the line height just to the tooltip I added (We need the space for the cloud env image in the new tooltip)
However - 1.5 line height is a bit more readable, and consistency is nice in my opinion.

Should we have a conversation with someone about it?

Reply:
I would suggesting pinging UX to see if they are OK with changing this in all of our tooltips.